api/tests/integration/infrastructure/repositories/schooling-registration-repository_test.js

```diff
@@ -1379,78 +1390,104 @@ describe('Integration | Infrastructure | Repository | schooling-registration-rep
+          userId: user.id,
+          nationalStudentId: schoolingRegistration.nationalStudentId,
+          organizationId: fakeOrganizationId,
+        });
 
-      // when
-      const error = await catchErr(schoolingRegistrationRepository.reconcileUserByNationalStudentIdAndOrganizationId)({
-        userId: user.id,
-        nationalStudentId: schoolingRegistration.nationalStudentId,
-        organizationId: fakeOrganizationId,
+        // then
+        expect(error).to.be.instanceof(UserCouldNotBeReconciledError);
       });
 
-      // then
-      expect(error).to.be.instanceof(UserCouldNotBeReconciledError);
-    });
+      it('should return an error when we don’t find the schoolingRegistration for this nationalStudentId to update', async () => {
+        // given
+        const fakeNationalStudentId = 1;
 
-    it('should return an error when we don’t find the schoolingRegistration for this nationalStudentId to update', async () => {
-      // given
-      const fakeNationalStudentId = 1;
+        // when
+        const error = await catchErr(schoolingRegistrationRepository.reconcileUserByNationalStudentIdAndOrganizationId)({
+          userId: user.id,
+          nationalStudentId: fakeNationalStudentId,
+          organizationId: organization.id,
+        });
 
-      // when
-      const error = await catchErr(schoolingRegistrationRepository.reconcileUserByNationalStudentIdAndOrganizationId)({
-        userId: user.id,
-        nationalStudentId: fakeNationalStudentId,
-        organizationId: organization.id,
+        // then
+        expect(error).to.be.instanceof(UserCouldNotBeReconciledError);
       });
 
-      // then
-      expect(error).to.be.instanceof(UserCouldNotBeReconciledError);
-    });
+      it('should return an error when the userId to link don’t match a user', async () => {
+        // given
+        const fakeUserId = 1;
 
-    it('should return an error when the userId to link don’t match a user', async () => {
-      // given
-      const fakeUserId = 1;
+        // when
+        const error = await catchErr(schoolingRegistrationRepository.reconcileUserByNationalStudentIdAndOrganizationId)({
+          userId: fakeUserId,
+          nationalStudentId: schoolingRegistration.nationalStudentId,
+          organizationId: organization.id,
+        });
 
-      // when
-      const error = await catchErr(schoolingRegistrationRepository.reconcileUserByNationalStudentIdAndOrganizationId)({
-        userId: fakeUserId,
-        nationalStudentId: schoolingRegistration.nationalStudentId,
-        organizationId: organization.id,
+        // then
+        expect(error).to.be.instanceof(UserCouldNotBeReconciledError);
       });
+    });
 
-      // then
-      expect(error).to.be.instanceof(UserCouldNotBeReconciledError);
+    context('when the schoolingRegistration is disabled', () => {
+      it('should return an error when the schoolingRegistration is disabled', async () => {
```

Comment:
```suggestion
      it('should return an error', async () => {
```
Le when est redondant avec le contexte